keyboards/adpenrose/kintsugi/keymaps/default/config.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2021 adpenrose
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ #pragma once
+
+/* Define custom font */
+#define OLED_FONT_H "keyboards/adpenrose/kintsugi/lib/kintsugifont.c"
```

Comment:
Please move this to post_config.h, and use: 

```c
#ifndef OLED_FONT_H
#    define OLED_FONT_H "lib/kintsugifont.c"
#endif
```

Reply:
Do I need to create a post_config.h file inside my keyboard root? In the official documentation it states that OLED_FONT_H should be placed inside the config.h file. Here's a link to that info: https://docs.qmk.fm/#/feature_oled_driver?id=basic-configuration

Reply:
post_config.h would be ideal, yes.  Though, it's newer.